hugo/data/footer.yml

```diff
@@ -39,7 +39,7 @@ networks:
     url: https://github.com/src-d
     image: img/icons/social/github.svg
   - title: medium
-    url: https://medium.com/source-d
+    url: https://medium.com/sourcedtech
```

Comment:
what's going to happen with the old https://medium.com/source-d that already has content? Is it going to be moved to the new one?

Reply:
It has only one post, but yeah, we could move it to the new one.